app/agents/voice/automatic/features/llm_wrapper.py

```
@@ -23,76 +23,95 @@ def __init__(self, llm_service):
+                                    params.arguments = final_args
+                                
+                                # Execute the original function
+                                result = await function(params)
+                                
+                                # Add success message
+                                success_msg = hitl_manager.generate_success_message(name, final_args)
+                                if hasattr(params, 'result_callback') and result_callback:
+                                    if isinstance(result, str):
+                                        enhanced_result = f"{result}\n\n{success_msg}"
+                                    else:
+                                        enhanced_result = f"{str(result)}\n\n{success_msg}"
```

Ruff (0.12.2)

Use explicit conversion flag

Replace with conversion flag

(RUF010)